flaml/integrations/oai/completion.py

```diff
@@ -213,7 +213,8 @@ def eval(cls, config: dict, prune=True, eval_only=False):
         prune_hp = getattr(cls, "_prune_hp", "n")
         metric = cls._metric
         config_n = config.get(prune_hp, 1)  # default value in OpenAI is 1
-        max_tokens = config.get("max_tokens", 16)  # default value in OpenAI is 16
+        max_tokens = config.get("max_tokens", 2048)  # most OpenAI models
+        # have a context length of 2048 tokens (except for the newest models, which support 4096)
```

Comment:
I'm not sure about this. When users don't specify max_tokens, the OpenAI's API default value is 16. Why do we change the default value which becomes inconsistent with OpenAI's API?